geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapred/GeoMesaInputFormat.scala

```diff
@@ -159,11 +161,15 @@ class GeoMesaInputFormat extends InputFormat[Text, SimpleFeature] {
   override def getSplits(job: JobConf, numSplits: Int) = {
     init(job)
     val accumuloSplits = delegate.getSplits(job, numSplits)
-    // try to create 2 mappers per node - account for case where there are less splits than shards
-    val groupSize = Math.max(numShards * 2, accumuloSplits.length / (numShards * 2))
+    // fallback on creating 2 mappers per node if desiredSplits is unset.
+    // Account for case where there are less splits than shards
+    val groupSize =  if (desiredSplitCount > 1) {
```

Comment:
I don't think there's a situation where we would want a user asking for one partition. That seems dangerous.